packages/renderer/src/lib/task-manager/TaskManagerActions.spec.ts

```diff
@@ -0,0 +1,60 @@
+/**********************************************************************
```

Comment:
Could we add some depth to the `lib/task-manager` by adding some subfolders ?

- `lib/task-manager/actions`
- `lib/task-manager/table`
etc.

That's a lot of files, and we can probably ease the navigation by creating some subfolders

Reply:
fixed in the latest commit